RecoVertex/Configuration/python/RecoVertex_cff.py

```diff
@@ -85,10 +85,11 @@
 from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
 phase2_timing.toReplaceWith(vertexrecoTask, _phase2_tktiming_vertexrecoTask)
 
-from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
-phase2_timing_layer.toReplaceWith(vertexrecoTask, _phase2_tktiming_layer_vertexrecoTask)
-phase2_timing_layer.toReplaceWith(unsortedOfflinePrimaryVertices4D, unsortedOfflinePrimaryVertices4DwithPID.clone())
-phase2_timing_layer.toReplaceWith(offlinePrimaryVertices4D, offlinePrimaryVertices4DwithPID.clone())
-phase2_timing_layer.toReplaceWith(offlinePrimaryVertices4DWithBS, offlinePrimaryVertices4DwithPIDWithBS.clone())
-phase2_timing_layer.toModify(offlinePrimaryVertices4D, vertices = "unsortedOfflinePrimaryVertices4D", particles = "trackRefsForJetsBeforeSorting4D")
-phase2_timing_layer.toModify(offlinePrimaryVertices4DWithBS, vertices = "unsortedOfflinePrimaryVertices4D:WithBS", particles = "trackRefsForJetsBeforeSorting4D")
+from Configuration.Eras.Modifier_phase2_timing_layer_tile_cff import phase2_timing_layer_tile
+from Configuration.Eras.Modifier_phase2_timing_layer_bar_cff import phase2_timing_layer_bar
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toReplaceWith(vertexrecoTask, _phase2_tktiming_layer_vertexrecoTask)
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toReplaceWith(unsortedOfflinePrimaryVertices4D, unsortedOfflinePrimaryVertices4DwithPID.clone())
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toReplaceWith(offlinePrimaryVertices4D, offlinePrimaryVertices4DwithPID.clone())
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toReplaceWith(offlinePrimaryVertices4DWithBS, offlinePrimaryVertices4DwithPIDWithBS.clone())
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toModify(offlinePrimaryVertices4D, vertices = "unsortedOfflinePrimaryVertices4D", particles = "trackRefsForJetsBeforeSorting4D")
+(phase2_timing_layer_tile | phase2_timing_layer_bar).toModify(offlinePrimaryVertices4DWithBS, vertices = "unsortedOfflinePrimaryVertices4D:WithBS", particles = "trackRefsForJetsBeforeSorting4D")
```

Comment:
the last two changes look like an overkill and ```phase2_timing_layer``` should be used instead.

Reply:
The module era modifiers should be made in the same file where the modules are defined.
So, only the ```toReplaceWith(vertexrecoTask``` belongs in this file.
I recall some 11th hour back-and-forth with the previous PR and can't recall if there was a real problem or just some greedy copy-paste to fix some last minute problems.

Reply:
There is a clone() here to avoid confusing changes to the original module, so technically the modules being modified ARE defined here.

Reply:
And ok, yes, for the last two lines, either way would be functional, I find this way less confusing, because in the phase2_timing_layer case, the parameters are already set to these values so the replace is redundant.

Reply:
> And ok, yes, for the last two lines, either way would be functional, I find this way less confusing, because in the phase2_timing_layer case, the parameters are already set to these values so the modify is redundant.

Oh. Good.